okpt/io/config/schemas/opensearch.yml

```diff
@@ -0,0 +1,17 @@
+# defined using the cerberus validation API
+# https://docs.python-cerberus.org/en/stable/index.html
+
+index_settings:
```

Comment:
Ah this could be confusing, because the index mapping is different from the settings. Maybe change this to `index_spec`.

Reply:
Done